HealthMonitoring.SelfHost/HealthMonitoring.SelfHost.csproj

```diff
@@ -195,6 +196,7 @@
     <EmbeddedResource Include="Content\Scripts\advanced-filters.js" />
     <EmbeddedResource Include="Content\Scripts\advanced-directives.js" />
     <EmbeddedResource Include="Content\Templates\wg-multiselect.html" />
+    <Content Include="Content\Scripts\angular-mocks.js" />

```

Comment:
the production code should not have mocks in it. It looks like it should be more in tests project no?